concise-mid-tag.cddl

```diff
@@ -131,9 +131,15 @@ tagged-int-type = #6.551(int)
 ueid-type = bytes .size 33
 tagged-ueid-type = #6.550(ueid-type)
 
+;
+; mkey type definitions
+;
 $measured-element-type-choice /= tagged-oid-type
 $measured-element-type-choice /= tagged-uuid-type
+; The $measured-element-type-choice uint is intended for time series mval data where lower uint
```

Comment:
I would prefer an open ended sentence, like could be used for time series data

Reply:
You're saying "could be" is better than "intended"?

Reply:
`can be used for time series`